css-three-column-layout/styles.css

```diff
@@ -0,0 +1,55 @@
+.container {
+  max-width: 1500px;
+  margin: 0 auto;
+  flex-wrap: wrap;
```

Comment:
Your `.container` doesn't have `display: flex`, so this isn't doing anything, so it should be deleted